src/components/button-panel.js

```diff
@@ -1,39 +1,44 @@
 /* eslint-disable jsx-quotes */
+/* eslint-disable react/prop-types */
+
 import React from 'react';
 import Button from './button';
 
-const ButtonPanel = () => (
-  <div>
-    <div className='row btn-group'>
-      <Button name='AC' btn='btn' color='grey' />
-      <Button name='+/-' btn='btn' color='grey' />
-      <Button name='%' btn='btn' color='grey' />
-      <Button name='÷' btn='btn' />
-    </div>
-    <div className='row btn-group'>
-      <Button name='7' btn='btn' color='grey' />
-      <Button name='8' btn='btn' color='grey' />
-      <Button name='9' btn='btn' color='grey' />
-      <Button name='X' btn='btn' />
-    </div>
-    <div className='row btn-group'>
-      <Button name='4' btn='btn' color='grey' />
-      <Button name='5' btn='btn' color='grey' />
-      <Button name='6' btn='btn' color='grey' />
-      <Button name='-' btn='btn' />
-    </div>
-    <div className='row btn-group'>
-      <Button name='1' btn='btn' color='grey' />
-      <Button name='2' btn='btn' color='grey' />
-      <Button name='3' btn='btn' color='grey' />
-      <Button name='+' btn='btn' />
-    </div>
-    <div className='row btn-group'>
-      <Button name='0' btn='btn' wide color='grey' />
-      <Button name='.' btn='btn' color='grey' />
-      <Button name='=' btn='btn' />
+const ButtonPanel = ({ clickHandler }) => {
+  const handleClick = buttonName => clickHandler(buttonName);
+  return (
+    <div>
+      <div className='row btn-group'>
+        <Button name='AC' btn='btn' color='grey' clickHandler={handleClick} />
+        <Button name='+/-' btn='btn' color='grey' clickHandler={handleClick} />
+        <Button name='%' btn='btn' color='grey' clickHandler={handleClick} />
+        <Button name='÷' btn='btn' clickHandler={handleClick} />
+      </div>
+      <div className='row btn-group'>
+        <Button name='7' btn='btn' color='grey' clickHandler={handleClick} />
+        <Button name='8' btn='btn' color='grey' clickHandler={handleClick} />
+        <Button name='9' btn='btn' color='grey' clickHandler={handleClick} />
+        <Button name='X' btn='btn' clickHandler={handleClick} />
```

Comment:
[Optional]

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.